packages/vscode-extension/src/builders/buildAndroid.ts

```diff
@@ -76,14 +76,38 @@ export async function buildAndroid(
   outputChannel: OutputChannel,
   progressListener: (newProgress: number) => void
 ): Promise<AndroidBuildResult> {
+  const { buildScript, env, android } = getLaunchConfiguration();
+
+  if (buildScript) {
+    const buildProcess = cancelToken.adapt(
+      exec(buildScript.name, ["android", ...(buildScript.args ?? [])])
+    );
+    let apkPath: string | undefined;
+    lineReader(buildProcess).onLineRead((line) => {
+      apkPath = line.trim();
+    });
+
+    await buildProcess;
```

Comment:
this may also throw in which case I don't know what  the expected result to the user will be? Maybe we should handle it separately and give some more descriptive error